src/styles/zoomwindow.scss

```diff
@@ -0,0 +1,53 @@
+@import "variables.scss";
+
+.zoomwindow {
+  background: white;
+  position: fixed;
+  left: 5px;
+  top: 5px;
+  z-index: 1;
+  width: 25px;
+  height: 25px;
+  padding-right:40px;
+  padding-left:5px;
+  align-content: center;
+  border-width: 3px;
+  border-style: solid;
+  border-color: $navbar-color;
+  margin: 3px;
+  margin-top: 50px;
+
+  &--hide {
+   opacity: 0;
+   animation-name: fadeOutOpacity;
+   animation-iteration-count: 1;
+   animation-timing-function: ease-out;
+   animation-duration: 1s;
+  }
+  &--show {
+    opacity: 1;
+    // animation-name: fadeInOpacity;
+    // animation-iteration-count: 1;
+    // animation-timing-function: ease-out;
+    // animation-duration: 1s;
+  }
+
+
+}
+
+@keyframes fadeInOpacity {
+	0% {
```

Comment:
you'll remove these after using `transition` anyways, but make sure to use 2 space tab width for frontend code